lib/components/common/timeline/list_timeline/widgets/list_timeline.dart

```diff
@@ -93,6 +102,16 @@ class _TopRow extends StatelessWidget {
               icon: const Icon(CupertinoIcons.refresh),
               onTap: () => bloc.add(const RequestListTimeline()),
             ),
+            defaultHorizontalSpacer,
+            if(list != null)
+              HarpyButton.raised(
+                padding: config.edgeInsets,
+                elevation: 0,
+                backgroundColor: theme.cardTheme.color,
+                icon: const Icon(CupertinoIcons.info),
+                onTap: () => 
+                  app<HarpyNavigator>().pushListMembersScreen(list: list!),
```

Comment:
```suggestion
                onTap: () => app<HarpyNavigator>().pushListMembersScreen(listId: listId, name : name),
```

(The navigator method and the ListMembersScreen arguments need to be updated accordingly, of course)